package.json

```diff
@@ -2,13 +2,16 @@
   "name": "security-wg",
   "version": "1.0.0",
   "scripts": {
-    "test": "node tools/vuln_valid/index.js"
+    "test": "node tools/vuln_valid",
+    "report": "node tools/reporter"
```

Comment:
can you add a README with usage instructions?

Reply:
ofcourse!

Reply:
@vdeturckheim added initial readme